MoltenVK/MoltenVK/GPUObjects/MVKRenderPass.mm

```diff
@@ -138,6 +138,12 @@
 
 	_mtlDummyTex = nil;
 	if (caUsedCnt == 0 && dsRPAttIdx == VK_ATTACHMENT_UNUSED) {
+        if ([mtlRPDesc respondsToSelector: @selector(defaultRasterSampleCount)]) {
+            // We support having no attachments.
+            mtlRPDesc.defaultRasterSampleCount = 1;
```

Comment:
This line needs to be wrapped in a `#if MVK_MACOS_OR_IOS`, to avoid compilation errors on `tvOS`, where  `defaultRasterSampleCount ` is not supported at all. See examples elsewhere.